app/views/form/review.html.erb

```diff
@@ -21,14 +21,18 @@
             <h3 class="x-govuk-summary-card__title"><%= subsection.label %></h3>
           </div>
           <div class="x-govuk-summary-card__body">
-            <dl class="govuk-summary-list">
+            <%= govuk_summary_list do |summary_list| %>
```

Comment:
Same comment here. This also looks similar enough to above to potentially be extracted into a partial?

Reply:
> This also looks similar enough to above to potentially be extracted into a partial?

Ah, good catch. Am now a using a partial for the entire list, which means passing around fewer params. 👌